PacketDotNet/IpPacket.cs

```diff
@@ -43,7 +43,7 @@ public abstract class IpPacket : InternetPacket
 #endif
 
         /// <summary>
-        /// The default time to live value for Ip packets being constructed
```

Comment:
Quite frankly, I'm not sure what changed here, maybe VS did some line ending changes after I've opened the file.